### PR TITLE
Hotfix/clearbit roles

### PR DESCRIPTION
**Pull request by unity**

### CODE REVIEW

File: packages/connectors/hull-clearbit/manifest.json [hunk 1]
```diff
@@ -1609,7 +1609,7 @@
       "description": "Hit `Enter` after every title to make a Tag out of it, Restrict your search by specifying criteria for the prospect's title. If left blank, search is carried out without restriction."
     },
     {
-      "name": "prospect_filter_role",
+      "name": "prospect_filter_roles",
       "title": "Filter for a certain Role",
```
Comment (SMK1085) [inline packages/connectors/hull-clearbit/manifest.json]: Can we make the title plural as well please

File: packages/connectors/hull-clearbit/manifest.json [hunk 2]
```diff
@@ -1644,7 +1644,7 @@
       "description": "Specify a certain role or leave blank to search without restriction."
     },
     {
-      "name": "prospect_filter_seniority",
+      "name": "prospect_filter_seniorities",
       "title": "Filter for specific Seniority",
```
Comment (SMK1085) [inline packages/connectors/hull-clearbit/manifest.json]: Same here, can we have plural please

Comment (unity) [reply packages/connectors/hull-clearbit/manifest.json]: Fixed

File: packages/connectors/hull-clearbit/manifest.json (hunk 1 above)
Comment (unity) [reply packages/connectors/hull-clearbit/manifest.json]: Fixed

**Pull request merged**